server/test/api/suppliers.js

```diff
@@ -0,0 +1,120 @@
+/* global describe, it, beforeEach */
+
+var chai = require('chai');
+var expect = chai.expect;
+var uuid    = require('node-uuid');
+
+var helpers = require('./helpers');
+helpers.configure(chai);
+
+/**
+* The /supplier  API endpoint
+*
+* This test suite implements full CRUD on the /supplier  HTTP API endpoint.
+*/
+describe('The /supplier  API endpoint', function () {
+  var agent = chai.request.agent(helpers.baseUrl);
+
+      // Supplier we will add during this test suite.
+  var supplier = {
+    uuid : uuid(),
```

Comment:
This should be `uuid.v4()`.  Calling `uuid()` directly [is deprecated](https://www.npmjs.com/package/node-uuid#uuidformat--buffer--offset).